### PR TITLE
Replace all :trac: with :issue: in docstrings

### PR DESCRIPTION
**Pull request by GiacomoPope**
Following the PR https://github.com/sagemath/sage/pull/37385, I have replaced all occurrences of `:trac:` with `:issue:`.

If changing so much in one PR is wrong and this should be done file by file while working on other functions feel free to close this PR.

### CODE REVIEW

**Comment by kwankyu** [comment]
Perhaps not to do in this PR, but I was thinking of changing "github issue" to just "issue". For example, 
```
implementation. By github issue #7797, there is a different implementation ...
``` 
is changed to
```
implementation. By issue #7797, there is a different implementation ...
``` 
We may redefine ``:issue:`` role to do this. What do you think?

**Comment by kwankyu** [comment]
> If changing so much in one PR is wrong ...

No problem.

**Review by kwankyu — APPROVED** [review]
Passes test. LGTM.

**Comment by GiacomoPope** [comment]
re: 

> We may redefine :issue: role to do this. What do you think?

Yeah, this could be done. Although there's probably a non-trivial amount of people who do not know what an "issue" is and "GitHub issue" might help newcomers better understand the docstrings from within Sage?

**Comment by kwankyu** [comment]
That applies only to the first day of their developer life. For all other days, "issue #7797" is short and simple. 

Perhaps  "Issue #7797" may be better that "issue #7797", if we want to emphasize that it is a github issue.

@mkoeppe ?

**Comment by GiacomoPope** [comment]
I have no strong opinions. Whatever is decided we can do in a new PR?

**Comment by kwankyu** [comment]
> I have no strong opinions. Whatever is decided we can do in a new PR?

I am collecting opinions. If a few collected opinions are supportive for the change, yes we can do it in a new PR.

If not clearly supportive, a poll may be necessary on sage-devel.

Thanks for your opinion.

**Comment by mantepse** [comment]
Please don't do "Github issue".  It is a lot of text with extremely little extra value.  I would hope that non-developers do not see such notices unless they are using deprecated functionality, in which it comes with a complete link.

**Comment by mkoeppe** [comment]
Yes, I also prefer "Issue #7797" (capitalized)

**Comment by vbraun** [comment]
merge conflict

**Comment by kwankyu** [comment]
@vbraun This PR is very likely to incur merge conflicts.

So could you consider making a beta only for this PR? That is, you release next beta, and then we rebase this PR to the beta, and then you immediately merge this PR and release a new beta.

**Comment by GiacomoPope** [comment]
@vbraun I see you have recently merged a few PR to develop, so I fixed the conflict and merged again. If these tests pass now then following @kwankyu's comment then maybe this can be squeezed in.

File: src/sage/schemes/elliptic_curves/hom_scalar.py [hunk 1]
```diff
@@ -81,8 +81,8 @@
     sage: zero(E.random_point())
     (0 : 1 : 0)
 
-Due to a bug (:issue:`6413`), retrieving multiplication-by-`m` maps
-when `m` is divisible by the characteristic currently fails::
+    Retrieving multiplication-by-`m` maps when `m` is divisible by the
+    characteristic also works (since :issue:`37096`)::
```
Comment (kwankyu) [inline src/sage/schemes/elliptic_curves/hom_scalar.py]: Unindent lines 84-85.

Comment (GiacomoPope) [reply src/sage/schemes/elliptic_curves/hom_scalar.py]: Thank you. I should not resolve merge conflicts using the GitHub website, too easy to make silly mistakes.

**Comment by kwankyu** [comment]
This will fix the merge conflicts:
```diff
diff --cc src/sage/rings/complex_arb.pyx
index 4ee2ec45015,31d1baf0f38..46c01da4eb6
--- a/src/sage/rings/complex_arb.pyx
+++ b/src/sage/rings/complex_arb.pyx
@@@ -4792,18 -4792,14 +4792,14 @@@ cdef class ComplexBall(RingElement)
  
              sage: n = CBF(1,1)
              sage: m = CBF(-2/3, 3/5)
-             sage: n.elliptic_pi_inc(CBF.pi()/2, m) # arb216
-             [0.8934793755173 +/- ...e-14] + [0.95707868710750 +/- ...e-15]*I
-             sage: n.elliptic_pi_inc(CBF.pi()/2, m) # arb218 - this is a regression, see :issue:28623
 -            sage: n.elliptic_pi_inc(CBF.pi()/2, m) # this is a regression, see :trac:28623
++            sage: n.elliptic_pi_inc(CBF.pi()/2, m) # this is a regression, see :issue:28623
              nan + nan*I
              sage: n.elliptic_pi(m)
              [0.8934793755173...] + [0.957078687107...]*I
  
              sage: n = CBF(2, 3/7)
              sage: m = CBF(-1/3, 2/9)
-             sage: n.elliptic_pi_inc(CBF.pi()/2, m) # arb216
-             [0.2969588746419 +/- ...e-14] + [1.3188795332738 +/- ...e-14]*I
-             sage: n.elliptic_pi_inc(CBF.pi()/2, m) # arb218 -  this is a regression, see :issue:28623
 -            sage: n.elliptic_pi_inc(CBF.pi()/2, m) # this is a regression, see :trac:28623
++            sage: n.elliptic_pi_inc(CBF.pi()/2, m) # this is a regression, see :issue:28623
              nan + nan*I
              sage: n.elliptic_pi(m)
              [0.296958874641...] + [1.318879533273...]*I
```

**Comment by kwankyu** [comment]
@vbraun Please merge this to 10.3.rc3.

**Pull request merged**